blueprints/sara2.py

```diff
@@ -0,0 +1,79 @@
+from flask import Blueprint, request, jsonify
+import requests
+
+sara2= Blueprint("sara2", __name__, url_prefix="/api/sara")
+
+@sara2.route('/user/<user_id>/todos', methods=["GET"])
+def todos_user(user_id):
+    response= requests.get(f"https://jsonplaceholder.typicode.com/users/{user_id}/todos")
+
+    if user_id is None:
+        return jsonify(error= "user_id is None")
+    
+    if int(response.status_code)//100 != 2:
+        return jsonify(error= "Invalid request")
+    
+    return response.json()
+
+@sara2.route('/user/<user_id>/posts', methods=["GET"])
+def posts_user(user_id):
+    response= requests.get(f"https://jsonplaceholder.typicode.com/users/{user_id}/posts")
+    
+    if user_id is None:
+        return jsonify(error= "user_id is None")
```

Comment:
When returning errors, you are dealing with a user who either uses JavaScript or doesn't know programming. The appropriate message is `user_id is empty` or `user_id is null` since `None` is `null` in JS.

No change required.